build.gradle

```diff
@@ -570,15 +619,15 @@ if (jdkVersion == '8') {
     tasks.jar.dependsOn tasks.shadowJar
 
     dependencies {
-      compile project(':iceberg-data')
-      compile project(':iceberg-orc')
+      implementation project(':iceberg-data')
+      implementation project(':iceberg-orc')
 
       testCompileOnly project(path: ':iceberg-data', configuration: 'testArtifacts')
       testCompileOnly project(path: ':iceberg-orc', configuration: 'testArtifacts')
     }
 
     shadowJar {
-      configurations = [project.configurations.compile, project.configurations.compileOnly, project.configurations.testCompileOnly]
```

Comment:
Why did this have a dependency on test compile deps ... that doesn't sound right.

Reply:
don't know why it had this dependency but I agree that it wasn't correct